core/src/test/java/io/confluent/rest/SslFactoryTest.java

```diff
@@ -0,0 +1,227 @@
+package io.confluent.rest;
+
+import org.apache.kafka.common.config.SslConfigs;
+import org.apache.kafka.common.config.types.Password;
+import org.apache.kafka.common.errors.InvalidConfigurationException;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.KeyStore;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class SslFactoryTest {
+  protected static final String CA1 = "-----BEGIN CERTIFICATE-----\n"
```

Comment:
Could these certificate and private key be generated by code? It is not ideal to have those static value stored here.

Reply:
Yes will look into it